packages/specification/prescriptions-api.yaml

```diff
@@ -0,0 +1,269 @@
+    | 500           | INTERNAL SERVER ERROR | The server has encountered a situation it does not know how to handle.                                                                |
+    | 504           | GATEWAY_TIMEOUT       | If your request has not been fully processed within 59 seconds, you will get this response. Retry the request with identical payload. |
+
+    For further details on common error codes, see [HTTP status codes](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes)
+
+x-spec-publication:
+  operation-order:
+    - operations:
+        - method: POST
+          path: /GetMyPrescriptions/Bundle
+
+paths:
+  /GetMyPrescriptions/Bundle:
+    get:
+      operationId: get-my-prescriptions-bundle
+      summary: Get prescriptions for the given user
+      description: |
+        ## Overview
+        Use this endpoint to get a Bundle containing a list of the user's prescriptions.
+      parameters:
+        - $ref: "#/components/parameters/BearerAuthorisation"
+        - $ref: "#/components/parameters/NhsNumber"
+        - $ref: "#/components/parameters/RequestID"
+      responses:
+        "200":
+          description: Successful retrieval.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/bundle-response"
+              examples:
+                prescriptions-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, containing one or more prescriptions.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success.json
+                prescriptions-not-present:
+                  description: A successful response to a GetMyPrescriptions/Bundle request, where no prescriptions were found.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/success-empty.json
+        "4XX":
+          description: Invalid request.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: "#/components/schemas/operation-outcome"
+              examples:
+                example:
+                  description: |
+                    An error response to a GetMyPrescriptions/Bundle request.
+                    Resource not found for the given NHS number.
+                  value:
+                    $ref: examples/GetMyPrescriptions/Bundle/error.json
+
+components:
+  parameters:
+    BearerAuthorisation:
+      in: header
+      name: Authorization
+      description: |
+        An [OAuth 2.0 bearer token](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis).
+      required: true
+      schema:
+        type: string
+        format: '^Bearer\ [[:ascii:]]+$'
+        example: "Bearer g1112R_ccQ1Ebbb4gtHBP1aaaNM"
+    NhsNumber:
```

Comment:
We don't want the NHS Number as a parameter. Our Apigee Proxy gets this for us from the Bearer token.

Reply:
Removed NhsNumber header.